### PR TITLE
Planning: Finish porting and enable all debug chart for open space planner

### PR DESCRIPTION
**Pull request by Capri2014**
Will optimize and re-org of the charts later.

### CODE REVIEW

File: modules/planning/on_lane_planning.cc [hunk 1]
```diff
@@ -705,13 +705,12 @@ void OnLanePlanning::AddPartitionedTrajectory(
   }
   chart->set_title("Open Space Partitioned Trajectory");
   // has to define chart boundary first
-  auto chosen_trajectory =
-      open_space_debug.chosen_trajectory().trajectory()[0];
+  auto chosen_trajectory = open_space_debug.chosen_trajectory().trajectory()[0];
```
Comment (xiaoxq) [inline modules/planning/on_lane_planning.cc]: `const auto&`?

File: modules/planning/on_lane_planning.cc [hunk 2]
```diff
@@ -746,5 +745,160 @@ void OnLanePlanning::AddPartitionedTrajectory(
...
+
+  // auto smoothed_trajectory = open_space_debug.smoothed_trajectory();
+  auto* speed_profile = chart->add_line();
+  speed_profile->set_label("Speed Profile");
+  for (const auto& point : trajectory_pb->trajectory_point()) {
+    auto* point_debug = speed_profile->add_point();
+    point_debug->set_x(point.relative_time() +
+                       trajectory_pb->header().timestamp_sec());
+    if (trajectory_pb->gear() == canbus::Chassis::GEAR_DRIVE)
+      point_debug->set_y(point.v());
```
Comment (xiaoxq) [inline modules/planning/on_lane_planning.cc]: Add brackets to all scopes.

File: modules/planning/on_lane_planning.cc (hunk 1 above)
Comment (Capri2014) [reply modules/planning/on_lane_planning.cc]: Done.

File: modules/planning/on_lane_planning.cc (hunk 2 above)
Comment (Capri2014) [reply modules/planning/on_lane_planning.cc]: Done.

File: modules/planning/on_lane_planning.cc [hunk 3]
```diff
@@ -745,5 +746,161 @@ void OnLanePlanning::AddPartitionedTrajectory(
...
+  auto* speed_profile_properties = speed_profile->mutable_properties();
+  (*speed_profile_properties)["borderWidth"] = "2";
+  (*speed_profile_properties)["pointRadius"] = "0";
+  (*speed_profile_properties)["lineTension"] = "0";
+  (*speed_profile_properties)["fill"] = "false";
+  (*speed_profile_properties)["showLine"] = "true";
+}
+
+void OnLanePlanning::AddPublishedSpeed(ADCTrajectory* const trajectory_pb,
+                                       planning_internal::Debug* debug_chart) {
```
Comment (xiaoxq) [inline modules/planning/on_lane_planning.cc]: Accept const reference as input. Don't let others break you with a NULL or wild pointer.

Comment (Capri2014) [reply modules/planning/on_lane_planning.cc]: Done with other minor cleaning up to avoid confusion.

**Pull request merged**